src/stream.rs

```diff
@@ -106,7 +106,7 @@ pub struct Utf8LossyDecoder<Sink, A=NonAtomic>
           A: Atomicity
 {
     decoder: utf8::Decoder,
-    sink: Sink,
+    pub inner_sink: Sink,
```

Comment:
They are different types: `Utf8LossyDecoder` has that struct field directly, but `LossyDecoder` (which supports encodings other than UTF-8) contains an enum.